addons/website_event_track_quiz/i18n/website_event_track_quiz.pot

```diff
@@ -411,7 +411,7 @@ msgstr ""
 
 #. module: website_event_track_quiz
 #: model_terms:ir.ui.view,arch_db:website_event_track_quiz.leaderboard_search_bar
-msgid "Search courses"
+msgid "Search users"
```

Comment:
Don't remove the old translation but add a new one. That way the old one is always available for people that don't upgrade their template, and the new one will be used for people that upgrade or create a new db in 14.

Reply:
hi @tde-banana-odoo ,
updated the translation